src/classes/AwsProvider.ts

```diff
@@ -84,6 +88,17 @@ export class AwsProvider {
         this.serverless.service.setFunctionNames(this.serverless.processedInput.options);
     }
 
+    setVpcConfig(securityGroup: AwsCfInstruction, subnets: AwsCfInstruction[]): void {
```

Comment:
This should probably error if there's already a VPC setup?

Reply:
Yes, it should. And we could handle the "2 VPC constructs" case here too.